app/src/main/java/org/greenstand/android/TreeTracker/capture/TreeCaptureViewModel.kt

```diff
@@ -45,8 +45,7 @@ class TreeCaptureViewModel(
     }
 
     suspend fun captureLocation() {
-        _state.value = _state.value?.copy(isGettingLocation = true)
-//        _state.value = _state.value?.copy(convergencePercentage = treeCapturer.getConvergencePercentage())
+        _state.value = _state.value?.copy(isGettingLocation = true, convergencePercentage = locationDataCapturer.percentageConvergence)
```

Comment:
@Elforama @ZavenArra this is the only issue left with this PR. The locationDataCapturer.percentageConvergence returns only the default value and doesn't update the state when tracking. But from the logs I can see the value been updated. Any tip on how to make this work

Reply:
Where are you putting the logs that it gets updated.

Reply:
You're just reading a float, to get the value updated you need to listen to it. You'll need to add a convergence listener to the class that updates the value, then you can listen to the value to update the state.